app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -90,3 +90,4 @@ CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
 
 CONFIG_PROBE=y
 CONFIG_PROBE_DMA_MAX=2
+CONFIG_LOG_TIMESTAMP_64BIT=y
```

Comment:
I would suggest config change as the third patch for enabling, and the second for code changes.

Reply:
@RanderWang ack - this will make it more bisectable.

Reply:
done